Update GCP permissions for PX 2.2 (#436)
portworx/pxdocs: content/portworx-install-with-kubernetes/cloud/gcp/shared/prerequisites.md

Before installing _Portworx-Enterprise_, make sure your environment meets the following requirements:

> * **Image type**: Portworx is supported on GKE clusters provisioned on [Ubuntu Node Images](https://cloud.google.com/kubernetes-engine/docs/node-images). So it is important to specify the node image as **Ubuntu** when creating clusters.

Reviewer: @harsh-px The passive voice in this sentence is driving me crazy. Here's a rewrite, but did I alter the meaning from the original? 
```suggestion
* **Image type**: Only GKE clusters provisioned on [Ubuntu Node Images](https://cloud.google.com/kubernetes-engine/docs/node-images) support Portworx. You must specify the **Ubuntu** node image when you create clusters. 
```
